src/main/webapp/app/markdown-editor/commands/bold.command.ts

```diff
@@ -0,0 +1,28 @@
+import { Command } from './command';
+
+export class BoldCommand extends Command {
+
+        buttonIcon =  'bold';
+        buttonTranslationString = 'arTeMiSApp.multipleChoiceQuestion.editor.bold';
+
+        /**
+         * @function execute
+         * @desc Make/Remove bold text
+         *       1. Check if the selected text includes (**)
+         *       2. If included reduce the selected text by these elements and replace the selected text by textToAdd
+         *       3. If not included, add (**) before and after the selected text and insert them into the editor
+         *       4. Bold markdown appears
+         */
+    execute(): void {
+        const chosenText = this.getSelectedText();
+        let textToAdd = '';
```

Comment:
See last comment about the hoisting, it would also be good if the naming is consistent (chosenText vs. selectedText).

Reply:
Cool!